### PR TITLE
Change the generated release template to docs/README.md

### PR DESCRIPTION
**Pull request by zmerlynn**

### CODE REVIEW

**Comment by zmerlynn** [comment]
cc @thockin @erictune 

I also changed the first couple of releases on the page, but not many.

File: build/build-official-release.sh [hunk 1]
```diff
@@ -99,7 +99,7 @@ echo "     with the ${KUBE_RELEASE_VERSION} tag. Mark it as a pre-release."
 echo "  3) Upload the ${KUBE_BUILD_DIR}/kubernetes.tar.gz to GitHub"
 echo "  4) Use this template for the release:"
 echo ""
-echo "## [Documentation](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/docs)"
+echo "## [Documentation](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/docs/README.md)"
 echo "## [Examples](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/examples)"
```
Comment (thockin) [inline build/build-official-release.sh]: same change here?

Comment (zmerlynn) [inline build/build-official-release.sh]: I looked at that page, and it doesn't have as strong a TOC as the docs page: https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/README.md

It's mostly useless to send users there until that's true.

**Comment by thockin** [comment]
LGTM, Jenkins and shippable both look like flakes, OK to merge without those

**Comment by zmerlynn** [comment]
@k8s-bot test this

**Comment by rjnagal** [comment]
one shippable run passed, integration test timed out in the other one. Timeout isn't caused by doc change, merging :)

**Pull request merged**